app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,10 +26,10 @@ export default function NoteTable({
 }) {
   const [showEditNote, setShowEditNote] = useState(false);
   const [selectedNote, setSelectedNote] = useState({});
-
+  const parentContainerRef = useRef(null);
   return (
     <>
-      <div className="w-full">
+      <div className="w-full notes-table-height" ref={parentContainerRef}>
```

Comment:
I am failing to understand how this `ref` is being useful here. Are we manipulating this `parentContainerRef` anywhere else within this component? What is the difference if we don't pass in this `ref` over here?

Reply:
It is working without the ref. I'll remove it. Thanks.